templates/master/00-master/baremetal/files/baremetal-haproxy.yaml

```diff
@@ -7,7 +7,7 @@ contents:
     apiVersion: v1
     metadata:
       name: haproxy
-      namespace: openshift-openstack-infra
+      namespace: non-existent
```

Comment:
Q:  was this namespace supposed to be kni and missed? or is the openstack correct here (i see it's in bm dir)

Reply:
now kni namespace exists and it is used by "metal" platform. before #1123 it didn't exist and used by both openstack and metal platforms.
Therefore, in order to get things back to normal, we have temporarily place the pods in a non-existent namespace.